inferior-shell.asd

```diff
@@ -23,6 +23,6 @@
               (symbol-call :inferior-shell-test :test-suite)))
 
 (defsystem "inferior-shell/test"
-  :depends-on ("inferior-shell" "hu.dwim.stefil")
+  :depends-on ("inferior-shell" "fiveam")
   :description "testing inferior-shell"
```

Comment:
I'm not up-to-date with CL development anymore, but what's wrong with stefil? Is fiveam now somehow standard?

Reply:
I'm more familiar with `fiveam` as most libs I saw recently use that. I can fallback to `stefil` but it may take some time for me to learn its dsl. If you like `stefil` more, could you let me know how to break test-suites into subsuites?